ops/docker/rpc-proxy/docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [ -z "$SEQUENCER" ];then
+  echo "SEQUENCER env must be set, exiting"
+  exit 1
+fi
+
```

Comment:
We probably want to make sure that `ETH_CALLS_ALLOWED` is set

Reply:
added!